static/bidder-info/orbidder.yaml

```diff
@@ -10,3 +10,7 @@ capabilities:
     mediaTypes:
       - banner
       - native
+userSync:
+  redirect:
+    url: "https://orbidder.otto.de/pbs-usersync?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&redirect={{.RedirectURL}}"
```

Comment:
The redirect url is not setting the uid.
<img width="1670" alt="image" src="https://github.com/prebid/prebid-server/assets/87759626/dc67013f-77ae-4f9c-8f28-c393950dfc4d">

Reply:
@Sonali-More-Xandr , can you maybe provide the Request, that was send to our endpoint?
Was the macro included or not?
Should we add some code at our endpoint, if no macro is provided? Up to now, we only add a userid if the macro is provided.

Reply:
Here is the URL I am using

https://orbidder.otto.de/pbs-usersync?gdpr=&consent=&redirect=https%3A%2F%2Fib.adnxs.com%2Fsetuid%3Fbidder%3Dorbidder%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%5BODN_ID%5D